.github/pr-labeler.yml

```
@@ -1,3 +1,3 @@
-'Type: Bug / Error': 'bug/*'
-'Type: Enhancement': 'feature/*'
+'Type: Bug / Error': ['bug/*', fix/*]
+'Type: Enhancement': ['feature/*', 'feat/*']
 'Type: Other': 'other/*'
```

Sorry, I know that this isn't part of this PR, but is it worth adding the following branch names to `'Type: Other'`? I know I sometimes use all of these branch names.

```suggestion
'Type: Other': ['other/*', 'chore/*', 'test/*', 'refactor/*']
```

The CONTRIBUTING.md file says `chore/` at least:

https://github.com/mermaid-js/mermaid/blob/194ef202ac5df51056d98c14f13e9ca85f274c35/CONTRIBUTING.md?plain=1#L55-L59

Maybe we should also add a line for `'Area: Documentation': ['docs/*']`